data/campaigns/Winds_of_Fate/story/Morogor_Dialect.txt

```diff
@@ -129,7 +129,7 @@ Remembered as extremely dangerous foes by the glider and clasher castes.
 - (Exonym) The Horde -
 Also called The Devourers.
```

Comment:
This line should go too, it's the same idea as the removed lines. Also it makes no sense without the context from the removed lines.

Reply:
From @nemaara on [discord](https://discord.com/channels/231976805987385345/259976436490829825/887992202662412309):

> I have orcs down as "super eaters", capable of digesting a great array of food while humans can't digest cellulose structures like wood and some leaves, orcs would be able to in addition to being able to consume large quantities of meat and other things

If this is still planned as a biological characteristic for orcs then the exonym "Devourers" should remain in the drake lexicon (it's the reason I put it in there to begin with). Otherwise I would agree with removing it.

Reply:
That quote from @nemaara (2021-09-16, for looking it up in the IRC logs) is followed by a link to the Irdya Canon, which AFAIK is no longer planned, but would contradict other parts of WoF's new lore about The Horde being an ancient enemy of the drakes. https://wiki.wesnoth.org/IrdyaCanonSecretLore#Orcish_lore

Reply:
Oh, so they would be recent creations... Alright, I have no issue with removing this line then.

Reply:
For what it's worth, I removed the line on the basis of the word having precisely zero occurrences in the campaign.